### PR TITLE
It's an 'Instance' of apiserver

### PR DESCRIPTION
**Pull request by lavalamp**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controlplane/instance.go [hunk 1]
```diff
@@ -225,8 +225,8 @@ type EndpointReconcilerConfig struct {
 	Interval   time.Duration
 }
 
-// Master contains state for a Kubernetes cluster master/api server.
-type Master struct {
+// Instance contains state for a Kubernetes cluster api server instance.
```
Comment (hasheddan) [inline pkg/controlplane/instance.go:228]: Looks like this needs to be reflected in https://github.com/kubernetes/kubernetes/blob/66334f02e8c520df7973c397246da82cd4db2769/pkg/controlplane/instance_test.go#L202 also. Might be worth also updating var names there as well?

**Comment by wojtek-t** [comment]
```
/b/f/w/pkg/controlplane/instance_test.go:202:32: undefined: Master
```

Once fixed lgtm

**Comment by deads2k** [comment]
/lgtm
/hold

you have a compile error and a nit.

**Comment by lavalamp** [comment]
I got too busy to fix the compile error, but @spiffxp has got it in #95044

**Pull request closed without merging**